spec/intelligent_foods/resources/menu_spec.rb

```diff
@@ -24,4 +24,55 @@
       end
     end
   end
+
+  describe ".find" do
+    it "returns the menu" do
+      menu_id = "2023-01-01"
+      body = build_menu_response(menu_id: menu_id)
+      response = build_response(body: body)
+      stub_api_response response: response
+
+      result = IntelligentFoods::Menu.find(menu_id)
+
+      expect(result.id).to eq(menu_id)
+    end
+
+    it "assigns correct number of items" do
+      menu_id = "2023-01-01"
+      menu_items = stubbed_menu_items
+      expected_items_count = menu_items.size
```

Comment:
this is a little too mystery guest for my taste. can you tell the stubbed menu items how many items you want and then hard code the expected items count amount

Reply:
https://thoughtbot.com/blog/mystery-guest
https://thoughtbot.com/upcase/decks/5/flashcards/27?mode=reviewing